### PR TITLE
Correct a number of typos

### PR DESCRIPTION
**Pull request by martynassateika**
Fixes a number of typos in comments.

### CODE REVIEW

File: fuel/modules/fuel/controllers/Dashboard.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ public function ajax()
 			$this->load->helper('simplepie');
 			$this->load->module_model(FUEL_FOLDER, 'fuel_pages_model');
 			$this->load->module_model(FUEL_FOLDER, 'fuel_logs_model');
-			$vars['recently_modifed_pages'] = $this->fuel_pages_model->find_all_array(array(), 'last_modified desc', 10);
+			$vars['recently_modified_pages'] = $this->fuel_pages_model->find_all_array(array(), 'last_modified desc', 10);
```
Comment (martynassateika) [inline fuel/modules/fuel/controllers/Dashboard.php]: This is not in a comment, but I fixed the usage as well.

File: fuel/modules/fuel/views/dashboard_ajax.php [hunk 2]
```diff
@@ -35,11 +35,11 @@
 </div>
 <?php endif; ?>
 
-<?php if (!empty($recently_modifed_pages)) : ?>
+<?php if (!empty($recently_modified_pages)) : ?>
```
Comment (martynassateika) [inline fuel/modules/fuel/views/dashboard_ajax.php]: And here are the usages :-)

**Comment by daylightstudio** [comment]
Goodness... who wrote this thing (thanks for the contributions)?

**Comment by martynassateika** [comment]
Haha no worries :-) PhpStorm helped me see the typos so it didn't take long to fix.

**Pull request merged**